itou/www/signup/tests.py

```diff
@@ -728,6 +728,146 @@ def test_create_user_prescriber_without_org(self):
         subject = mail.outbox[0].subject
         self.assertIn("Confirmez votre adresse e-mail pour la Plateforme", subject)
 
+    @mock.patch(
+        "itou.utils.apis.api_entreprise.EtablissementAPI.get", return_value=(ETABLISSEMENT_API_RESULT_MOCK, None)
+    )
+    @mock.patch("itou.utils.apis.geocoding.call_ban_geocoding_api", return_value=BAN_GEOCODING_API_RESULT_MOCK)
+    def test_create_user_prescriber_with_same_siret_and_different_kind(
+        self, mock_api_entreprise, mock_call_ban_geocoding_api
+    ):
+        """
+        A user can create a new prescriber organization with an existing SIRET number,
+        provided that:
+        * the kind of the new organization is different from the existing one <-
+        * there is no duplicate of the (kind, siret) pair
+
+        f.i.:
```

Comment:
J'ai pas trouvé ce que ça veut dire. En ligne on mentionne `fuck it` et `factual information`. J'imagine que tu voulais dire `for instance`? Perso j'utilise `e.g.` (par exemple) et `i.e.` (c'est à dire) cf https://www.grammarly.com/blog/know-your-latin-i-e-vs-e-g/#:~:text=I.e.%20and%20e.g.%20are%20both,alternative%20English%20translation%20of%20i.e.

Reply:
C'était effectivement for instance, mais je mets aussi i.e.